moveit_ros/planning/rdf_loader/include/moveit/rdf_loader/rdf_loader.h

```diff
@@ -54,7 +55,7 @@ class RDFLoader
   /** @brief Default constructor
    *  @param robot_description The string name corresponding to the ROS param where the URDF is loaded; the SRDF is
    * assumed to be at the same param name + the "_semantic" suffix */
-  RDFLoader(const std::string& robot_description = "robot_description");
+  RDFLoader(std::shared_ptr<rclcpp::Node>& node, const std::string& robot_description = "robot_description");
```

Comment:
We should either pass the shared pointer of `node` as const reference or by value. In general I would prefer passing it as const reference, but I realize there are points in favor of both versions.